lib/round.rb

```diff
@@ -0,0 +1,86 @@
+class Round
+  attr_reader :deck, :turns, :current_card, :number_correct
+
+  def initialize(deck)
+    @deck = deck
+    @turns = []
+    @number_correct = 0
+
+  end
+
+  def current_card
+     deck.cards[0]
+  end
+
+
+
+  def take_turn(new_guess)
+     turn = Turn.new(new_guess, current_card)
+     @turns << turn
+     if new_guess == current_card.answer
+       @number_correct += 1
+     end
+     deck.cards.rotate!
```

Comment:
Nice job using this built-in method. Be careful when using `!` methods, as they can inadvertently change the base object you're working on, and may affect things later on down the road. But for now, it works great.